README.md

```diff
@@ -8,13 +8,31 @@ Small tool to convert an IAM Policy in JSON format into a Terraform [`aws_iam_po
 
 Download the latest binary from the [releases page](https://github.com/flosell/iam-policy-json-to-terraform/releases) and put it into your `PATH` under the name `iam-policy-json-to-terraform`
 
+    brew install iam-policy-json-to-terraform
+
 If you're a go developer and have your `GOPATH` defined and have added your `$GOPATH/bin` directory to your path, you can simply run this command.
 
     go get github.com/flosell/iam-policy-json-to-terraform
 
 ## Usage
```

Comment:
Thanks for extending the usage docs, looks helpful!

Reply:
My pleasure